apps/meteor/client/views/room/Header/Omnichannel/QuickActions/hooks/useQuickActions.tsx

```diff
@@ -311,19 +312,20 @@ export const useQuickActions = (): {
 	const canCloseRoom = usePermission('close-livechat-room');
 	const canCloseOthersRoom = usePermission('close-others-livechat-room');
 	const canPlaceChatOnHold = Boolean(!room.onHold && room.u && !(room as any).lastMessage?.token && manualOnHoldAllowed);
+	const isRoomActive = useIsRoomActive(room);
```

Comment:
`useIsRoomActive` It's not such a cool name, I think the name should represent exactly what the hook checks 'agent can interact with visitor room'